src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -201,21 +201,19 @@ public async Task InitializeUIAffinitizedServicesAsync(IAsyncServiceProvider asy
             solutionClosingContext.UIContextChanged += (_, e) => _solutionClosing = e.Activated;
 
             var openFileTracker = await OpenFileTracker.CreateAsync(this, asyncServiceProvider).ConfigureAwait(true);
-
-            // Update our fields first, so any asynchronous work that needs to use these is able to see the service.
-            using (await _gate.DisposableWaitAsync().ConfigureAwait(true))
-            {
-                _openFileTracker = openFileTracker;
-            }
-
             var memoryListener = await VirtualMemoryNotificationListener.CreateAsync(this, _threadingContext, asyncServiceProvider, _globalOptions, _threadingContext.DisposalToken).ConfigureAwait(true);
```

Comment:
I'm starting a separate conversation about whether this even still needs to exist anymore.